package.json

```diff
@@ -49,9 +49,10 @@
     "lodash.omit": "^4.5.0",
     "lodash.union": "^4.6.0",
     "lodash.uniqby": "^4.4.0",
-    "react": "17.0.1",
+    "react": "17.0.2",
     "react-intl": "5.24.6",
-    "react-native": "0.64.3",
+    "react-native": "0.65.2",
+    "react-native-codegen": "^0.0.7",
```

Comment:
It looks like in the template app, `react-native-codegen` goes in `devDependencies`. Can we similarly put it there?